getting-started.md

```diff
@@ -491,3 +491,111 @@ $ kubectl get secret folder-outputs
 NAME         TYPE     DATA   AGE
 vm-outputs   Opaque   1      18m
 ```
+
+# For Elastic Cloud
+
+### Apply Provider configuration
+
+To interact with the EC Terraform provider an API key is expected. Please see Terraform EC provider documentation for [generating API keys](https://registry.terraform.io/providers/elastic/ec/latest/docs).
+```shell
+$ export EC_API_KEY=xxx
+
+$ sh hack/prepare-ec-credentials.sh
+
+$ kubectl get secret -n vela-system
+NAME                                              TYPE                                  DATA   AGE
+ec-account-creds                                 Opaque                                1      52s
+
+$ kubectl apply -f examples/ec/provider.yaml
+provider.terraform.core.oam.dev/ec created
+```
+
+### Apply Terraform Configuration
+
+Apply Terraform configuration [configuration_hcl_ecproject.yaml](./examples/ec/configuration_hcl_ecdeployment.yaml) to provision a Elastic Cloud project.
+```yaml
+apiVersion: terraform.core.oam.dev/v1beta1
+kind: Configuration
+metadata:
+  name: ec-deployment
+spec:
+  hcl: |
+    required_providers {
+      ec = {
+        source  = "elastic/ec"
+        version = "0.2.1"
+      }
```

Comment:
This section might not be right. You can refer to the tf-native example.

Reply:
@mattkirby After fixed this section, it should work. Please rerun it end to end. Also please snapshot the deployment created by this controller on the EC cloud dashboard and paste it here. After that, this PR can be merged.

Reply:
Thanks, I added the missing section to getting-started and the yaml example.
Attached is a screenshot of the deployment after running the example configuration through the oam-terraform controller. Please let me know if you need any additional screenshots or information.
![es screen](https://user-images.githubusercontent.com/3004602/130265630-0cdf2e94-06f1-4aa8-b19a-e1328491fffd.png)